packages/skeleton-svelte/src/components/Toast/ToastProvider.svelte

```diff
@@ -1,133 +1,76 @@
 <script lang="ts">
-	import { setContext } from 'svelte';
-	import { flip } from 'svelte/animate';
-	import { fade } from 'svelte/transition';
-
-	import type { PlacementStyles, Toast, ToastProviderProps } from './types.js';
+	import { normalizeProps, useMachine } from '@zag-js/svelte';
+	import * as toast from '@zag-js/toast';
+	import Toast from './Toast.svelte';
+	import type { ToastProviderProps } from './types.js';
+	import { setToastContext } from './context.js';
 
 	const {
-		placement = 'bottom-end',
-		offset = '16px',
-		dismissLabel = '',
-		// Group
-		base = 'fixed flex flex-col items-end',
-		gap = 'gap-4',
-		zIndex = 'z-[888]',
-		classes = '',
 		// Toast
-		toastBase = 'card py-2 px-3 grid grid-cols-[1fr_auto] items-center',
-		toastPadding = 'py-2 px-3',
-		toastGap = 'gap-4',
-		toastShadow = 'shadow-xl',
-		toastClasses = '',
+		base = 'flex justify-between items-center gap-3',
+		width = 'min-w-xs',
```

Comment:
FYI: Broke these out to smaller style props. I now `-3` is uncommon, but we want these to feel a bit smaller and more compact.